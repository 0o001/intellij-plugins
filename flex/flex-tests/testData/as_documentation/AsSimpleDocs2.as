////////////////////////////////////////////////////////////////////////////////
//
//  Copyright (C) 3000 by me.
//
////////////////////////////////////////////////////////////////////////////////

package controls
{

// !!!!
[Exclude(name="maxHorizontalScrollPosition", kind="property")]

/**
 *  The CCC control is super special control.
 */
public class CCC {
  /**
   * Constructor
   */
    function CCC() {}

  /**
   * my super
   * func
   * <table>
   *   <tr>
   *       <th>Property</th>
   *       <th>Value</th>
   *   </tr>
   *   <tr>
   *       <td><code>bubbles</code></td>
   *       <td>false</td>
   *   </tr>
   * </table>
   *
   * doc1
   * doc2
   *
   * <TABLE width="100%"></TABLE>
   * doc3
   * doc4
   */
    function aaa(a:int = 1) {
      a<caret>aa()
    }
  }
}
