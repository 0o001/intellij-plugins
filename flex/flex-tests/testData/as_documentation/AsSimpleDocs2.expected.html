<div class='definition'><pre>controls.CCC.aaa(<br>    a?: int = 1)</pre></div><div class='content'><p> my super<p>
 func<p>
 <table><p>
   <tr>       <th>Property</th>       <th>Value</th>   </tr><p>
   <tr>       <td><code>bubbles</code></td>       <td>false</td>   </tr><p>
 </table><p>
<p>
 doc1<p>
 doc2<p>
<p>
 <TABLE width="100%"></TABLE><p>
 doc3<p>
 doc4</div><table class='sections'></table>